frontend/src/metabase/collections/components/BulkActions.jsx

```diff
@@ -46,11 +46,26 @@ const SelectionControls = ({
   size = 18,
 }) =>
   !hasUnselected ? (
-    <StackedCheckBox checked onChange={onSelectNone} size={size} />
+    <StackedCheckBox
+      checked
+      onChange={onSelectNone}
+      size={size}
+      aria-label="Select all items"
```

Comment:
This is not in the original PR but here I need to add this `aria-label`, as the current bulk actions UI (visible on master) includes it and the e2e test depends on it.